internal/system/agent/services.go

```diff
@@ -33,14 +33,14 @@ const (
 	RESTART = "restart"
 )
 
-func InvokeOperation(action string, services []string) bool {
+func InvokeOperation(action string, services []string) error {
 
 	// Loop through requested operation, along with respectively-supplied parameters.
 	for _, service := range services {
-		LoggingClient.Info("invoking operation %s on service %s", action, service)
+		LoggingClient.Info("invoking operation", "operation", action, "service", service)
```

Comment:
Similar comments (as earlier) apply here, etc., too, regarding multiple keys in logging.